node/tests/env0-deploy-cli.spec.js

```diff
@@ -61,6 +62,22 @@ describe("env0-deploy-cli", () => {
             })
         })
 
+        describe.each`
+        command
+        ${'-v'}
+        ${'--version'}
+        ${'version'}
+        `('when user asks to see the version with $command', ({ command }) => {
+            beforeEach(async () => {
+                jest.spyOn(console, 'log');
+                await mockOptionsAndRun({ command, rawArgs: [ command ]});
+            })
+
+            it('should show the proper version', () => {
+                expect(console.log).toBeCalledWith(version);
+            })
+        })
```

Comment:
You are missing `;` in some places here. 
Let's add linter please...

Reply:
It will be done as a part of the publish to npm task